src/share/poudriere/common.sh

```diff
@@ -2090,7 +2090,7 @@ _real_build_port() {
 		# Don't need to install if only making packages and not
 		# testing.
 		[ -n "${PORTTESTING}" ] && \
-		    install_order="${install_order} install-mtree install"
+		    install_order="${install_order} test install-mtree install"
```

Comment:
Well, they don't depend on install, they depend on stage, so, I'm not certain it's true.